package.json

```diff
@@ -24,9 +24,9 @@
   },
   "dependencies": {
     "pg": "^6.1.3",
-    "sequelize": "^3.30.2"
+    "sequelize": "3.30.2 - 4.2"
```

Comment:
Ah I see. Yeah I'd also agree that just a peer dep would be more semantically correct, at least, but I don't feel super strongly if this works (and I don't see why it wouldn't).